src/app/auth/auth.component.ts

```diff
@@ -173,9 +184,11 @@ export class AuthComponent implements OnInit, OnDestroy {
       flatMap(() => this.githubEventService.setLatestChangeEvent()),
     ).subscribe(() => {
       this.handleAuthSuccess();
+      this.logger.info('Successfully completed login process');
```

Comment:
The last line of `handleAuthSuccess()` technically already prints a successful login so I'm not sure if this would be like double logging it.

![image](https://user-images.githubusercontent.com/39243082/111020577-ee226f80-8401-11eb-9ad3-5da2e8af4065.png)

Reply:
I think you're  right, a bit redundant😅 I have removed it!